src/windows/player.c

```diff
@@ -264,6 +265,65 @@ void window_player_overview_show_group_dropdown(rct_window *w, rct_widget *widge
 	dropdown_set_checked(network_get_group_index(network_get_player_group(player)), true);
 }
 
+#define KICK_REASON_INDEX_FIRST STR_KICK_REASON_NO_REASON
+#define KICK_REASON_INDEX_LAST STR_KICK_REASON_OTHER_REASON
+#define KICK_REASON_INDEX_COUNT (KICK_REASON_INDEX_LAST + 1 - KICK_REASON_INDEX_FIRST)
+#define KICK2STRING(x) (KICK_REASON_INDEX_FIRST + x)
+#define STRING2KICK(x) (x - KICK_REASON_INDEX_FIRST)
```

Comment:
Once strings are added after other_reason, you won't be able to just insert new ones inbetween them, or you would be limited by the space you left. We will move away from fixed string IDs eventually and assuming sequential string IDs is not a good idea.

You will need to create an enum and an array, see keyboard shortcuts as an example.